### ISSUE TITLE
remove not working

### ISSUE THREAD

**Issue by xioxin** [typicode/lowdb#47]
```javascript
var low = require('lowdb');
low.mixin(require('underscore-db'));
var db = low('db.json');
db('file').insert({name: '1.jpg',dir: 'test',size: 1024});
console.log(db('file').value())
/*
[ { name: '1.jpg',
    dir: 'test',
    size: 1024,
    id: '97f5caa4-2fb2-48bf-bfe5-42b26735387d' } ]
*/
db('file').remove({name: '1.jpg',dir: 'test'});
console.log(db('file').value());
/*
[ { name: '1.jpg',
    dir: 'test',
    size: 1024,
    id: '97f5caa4-2fb2-48bf-bfe5-42b26735387d' } ]
*/
```

**Comment by typicode** [comment]
Thanks for letting me know.

There's a method name collision between lodash and underscore-db. They both have `.remove`.
I'll fix that.

However, for the moment, you can use `.remove` by passing an id:

```javascript
var id = db('file').insert({name: '1.jpg',dir: 'test',size: 1024}).id;
db('file').remove(id);
```

**Comment by typicode** [comment]
I've updated [underscore-db](https://github.com/typicode/underscore-db) API and released `v0.9.0`.

Your example should now work. And if you want to remove using an id, you can do so with the dedicated `removeById(id)` method from `underscore-db`.
